ansible/ci-build-collector.yml

```diff
@@ -8,10 +8,23 @@
     COLLECTOR_TAG: "{{ ansible_env.COLLECTOR_TAG }}"
 
   tasks:
-    - name: Build the collector image
-      community.general.make:
-        chdir: "{{ lookup('env', 'GITHUB_WORKSPACE') }}"
-        target: image
+    - block:
+        - name: Build the collector image
+          community.general.make:
+            chdir: "{{ lookup('env', 'GITHUB_WORKSPACE') }}"
+            target: image
+          register: build_result
+          # ensure this action is printed
+          tags: [print_action]
+      rescue:
+        - name: Output built log
+          debug:
+            msg: "{{ build_result.stdout }}"
```

Comment:
Do we maybe need `build_result.stderr` as well? I would assume compilation errors would go to that stream

Reply:
I just double checked this behaviour and if the build task fails above, we get all the stderr output printed by default, which actually makes me feel like this rescue block is unnecessary because we probably only care about the errors and not the full build log. Alternatively, if we also want the full build log, we can change this to `always` - WDYT?

Reply:
Sounds good, we can keep build log, should make it a little easier to understand what is going on during failures.